jsaddle/src/Language/Javascript/JSaddle/Object.hs

```diff
@@ -462,8 +462,8 @@ function f = do
     return $ Function object
 #endif
 
--- ^ Make a JavaScript function object that wraps a Haskell function.
--- Calls made to the function will be Asynchronous.
+-- | Make a JavaScript function object that wraps a Haskell function.
```

Comment:
Same here, the haddocks claim that `function` is async, and there is no haddock comment for `asyncFunction`.